primitives/src/market.rs

```diff
@@ -26,6 +26,8 @@ pub struct Market<AccountId, BlockNumber> {
     pub report: Option<Report<AccountId, BlockNumber>>,
     // The resolved outcome.
     pub resolved_outcome: Option<OutcomeReport>,
+    // See [`MarketDisputeMechanism`].
+    pub mdm: MarketDisputeMechanism<AccountId>,
```

Comment:
Isn't 0.1.3 going to be a new a clean release? If storage will be a concern, then it will also be necessary to write migrations for the Swaps, PM and Court pallets.

Reply:
AFAIK this release was planned to be usual runtime upgrade. It sounds to me like in that case, a huge effort is necessary to migrate the storage. I am also open to do a clean release and potentially even start a new testnet, but we'd have to discuss this with @lsaether

Reply:
Hum... Maybe sync the new release with the upcoming collator program?

Reply:
I think we can merge this once *types.json* was adjusted and `spec_version` and `transaction_version` were bumped. We can do the migrations in a follow-up PR, if it is really necessary. I think in this case, we might want to disobey our release rules in favor of avoiding having to write a bunch of migrations for a testnet that is at its end of life.